src/Microsoft.ML.FastTree/BoostingFastTree.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.ML.Trainers.FastTree
     public abstract class BoostingFastTreeTrainerBase<TArgs, TTransformer, TModel> : FastTreeTrainerBase<TArgs, TTransformer, TModel>
         where TTransformer : ISingleFeaturePredictionTransformer<TModel>
         where TArgs : BoostedTreeArgs, new()
-        where TModel : IPredictorProducing<Float>
+        where TModel : class
     {
```

Comment:
I guess I will have to replace this with a more restrictive new type in my PR.

Reply:
Since it is a marker interface, the type restriction is irrelevant. Also since all invocations must be internal to the code anyway, there is no particular need for any restriction.